<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecurityEditPage</class>
 <widget class="QWidget" name="SecurityEditPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelRubric">
     <property name="text">
      <string>The security mode of this pool determines which SSL/TLS protocol versions can be used to connect to servers in this pool.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonTLS">
     <property name="text">
      <string>TLS verification enabled (recommended)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonSSL">
     <property name="text">
      <string>Use legacy SSL protocol (less secure)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutWarning">
     <item>
      <widget class="QLabel" name="pictureBoxDisruption">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/icons/alert.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDisruption">
       <property name="text">
        <string>Changing this setting will require all hosts in the pool to restart their management services. This will cause temporary connection disruption.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
